README.rst

```diff
@@ -21,6 +21,18 @@
 
 Get coala to lint all your languages in your project with one tool and config!
 
+Introduction
+------------
+
+coala is a language independent analysis toolkit. It empowers developers
+to create rules which a project's code should conform to. coala takes care
+of showing these issues to users in a friendly manner, is versatile and
+can be used in any environment. Patches to automatically fix code will
+be managed too. coala has a set of official bears (plugins) to provide
+an out-of-the-box analysis functionality for many popular languages in
+addition to some generically applicable algorithms.
```

Comment:
Well @souravsingh please add those 2 links at the end of the paragraph, https://gist.github.com/Adrianzatreanu/cf334ae9f19976e2d5d7
https://gist.github.com/Adrianzatreanu/cf2d0c8b2ecd542a4860